Packs/ContentManagement/Scripts/SuggestBranchName/SuggestBranchName.py

```diff
@@ -1,16 +1,30 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 
-ATTEMPS = 10
+ATTEMPTS = 10
+
+
+def find_available_branch_azure_devops(pack_name: str):
+    response = demisto.executeCommand('azure-devops-branch-list', args={})
+    existing_branches = response[0].get("Contents", {}).get("value", [])
+    for i in range(1, ATTEMPTS + 1):
+        branch_name = f'{pack_name}_{i}'
+        branch_exists = any(
+            branch.get("name", "").endswith(branch_name)
+            for branch in existing_branches
+        )
+        if not branch_exists:
+            return f'refs/heads/{branch_name}'
+    raise DemistoException('Please enter a branch name.')
 
 
 def find_available_branch(pack_name: str, command_get_branch: str):
     branch_name = pack_name
-    for i in range(ATTEMPS):
+    for i in range(ATTEMPTS):
         if i > 0:
             branch_name = f'{pack_name}_{i}'
         status, get_branch_res = execute_command(command_get_branch, {'branch_name': branch_name}, fail_on_error=False)
-        if (not status) and ('Bad Request' or 'Branch not found' or 'Not Found' in get_branch_res):
+        if (not status) and (True):
```

Comment:
This is indeed a pre-existing bug, but changing to `and True` doesn't change much 😬 
Please open a bug ticket and assign to whomever wrote it, so they can fix it and add unit tests